docs/2.8.0/docs/canton/usermanual/identity_management.rst

```diff
@@ -514,132 +514,231 @@ as automatically determining the parameters for the ``authorize`` call.
     Please note that the ``participant.parties.enable`` macro will add the parties to the same namespace as the participant is in.
     It only works if the participant has authority over that namespace either by possessing the root or a delegated key.
 
-.. enterprise-only::
+.. _separate-party-migration:
 
-.. _offline-party-migration:
+Client Controlled Party
+~~~~~~~~~~~~~~~~~~~~~~~
 
-Migrate Party to Another Participant Node
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. enterprise-only::
 
 .. note::
-
-    The improved macros are available in Daml Enterprise 2.x starting with 2.8.1.
+    - The improved macros are available in Daml Enterprise 2.x starting with 2.8.1.
+    - Please note that party migration in 2.x comes with limitations. Please read the documentation carefully.
+    - The marcos only work with protocol version 4 or higher.
+    - The involved participants must be entirely quiet during the migration. Therefore, the migration can only happen during a maintanenace window of the domain where the rate is set to 0.
```

Comment:
```suggestion
    - The improved macros are available in Daml Enterprise 2.x as of release 2.8.1.
    - In 2.x, party migration has limitations. Please read the documentation carefully.
    - The macros work with protocol version 4 or later.
    - The involved participants must be entirely quiet during the migration. Therefore, the migration can only happen during a maintenance window of the domain where the rate is set to 0.
```

Reply:
This change includes applying these style guidelines:

https://developers.google.com/style/word-list#please
https://developers.google.com/style/word-list#higher

Reply:
fixed